hpedockerplugin/volume_manager.py

```diff
@@ -1432,7 +1432,12 @@ def _mount_volume(driver):
         # Determine if we need to mount the volume
         if vol_mount == volume.DEFAULT_MOUNT_VOLUME:
             # mkdir for mounting the filesystem
-            mount_dir = fileutil.mkdir_for_mounting(device_info['path'])
+            if self._hpepluginconfig.mount_prefix:
+                mount_prefix = self._hpepluginconfig.mount_prefix
```

Comment:
if 'mount_prefix' is per host then 'host_config.mount_prefix' must be used.

Reply:
Done. Fixed as per the review comment in the new commit.

Reply:
i will correct it. Thanks for catching that :-)